crates/turborepo-lib/src/shim.rs

```diff
@@ -407,11 +409,35 @@ impl RepoState {
                 self.spawn_local_turbo(&canonical_local_turbo, shim_args),
             ))
         } else {
-            try_check_for_updates(&shim_args, get_version());
+            let global_version = get_version();
+            try_check_for_updates(&shim_args, global_version);
+            debug!("Running command as global turbo");
+
+            // Absence of turbo.json is not an error per business logic.
```

Comment:
It's an error downstream in Go for monorepos, btw. Single package repos don't need turbo.json.

Reply:
Right, that was my meaning. I just mean that "we don't stop here."